source/config/featureFlagEnums.py

```diff
@@ -70,10 +70,12 @@ class AppsVolumeAdjusterFlag(DisplayStringEnum):
 	@property
 	def _displayStringLabels(self):
 		return {
-			# Translators: Label for applications volume adjuster in NVDA settings.
-			self.DISABLED: _("Disabled Applications volume adjuster"),
-			# Translators: Label for applications volume adjuster in NVDA settings.
-			self.ENABLED: _("Enabled Applications volume adjuster"),
+			# Translators: A choice disabling "application volume adjustment"
+			# in the audio settings panel.
+			self.DISABLED: _("No"),
+			# Translators: A choice enabling "application volume adjustment"
+			# in the audio settings panel.
+			self.ENABLED: _("Yes"),
```

Comment:
shouldn't this be disabled/enabled not no/yes? I think thats more common

Reply:
@seanbudd, if the option name is "Allow NVDA to modify the volume of other applications" or "Allow whatever"), Yes/No makes much more sense than Enabled/Disabled which is a bit confusing.

Once, and if, you agree with Yes/No, other things need to be changed in the code: variable names, other strings, etc. We need to double check that all is consistent with Yes/No.

Reply:
This should be `no`/`yes` to match "allow"